adapters/akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala

```diff
@@ -166,14 +167,17 @@ trait AkkaHttpAdapter {
       }
   }
 
+  private def emptyAuth[R](message: String): RIO[R, Option[Authorization]] = ZIO.succeed(Option.empty[Authorization])
+
   def makeWebSocketService[R, E](
     interpreter: GraphQLInterpreter[R, E],
     skipValidation: Boolean = false,
     enableIntrospection: Boolean = true,
     keepAliveTime: Option[Duration] = None,
     contextWrapper: ContextWrapper[R, GraphQLResponse[E]] = ContextWrapper.empty,
     queryExecution: QueryExecution = QueryExecution.Parallel,
-    wsChunkTimeout: Duration = 5.seconds
+    wsChunkTimeout: Duration = 5.seconds,
+    onInitAuthHeader: String => RIO[R, Option[Authorization]] = emptyAuth(_: String)
```

Comment:
I tried to pass inline a `_ => ZIO.succeed(Option.empty[Authorization])` here, but the compiler didn't like something about inferring the String